lib/web_ui/test/engine/semantics/semantics_multi_view_test.dart

```diff
@@ -0,0 +1,129 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+
+import '../../common/test_initialization.dart';
+import 'semantics_tester.dart';
+
+void main() {
+  internalBootstrapBrowserTest(() {
+    return testMain;
+  });
+}
+
+Future<void> testMain() async {
+  // TODO(yjbanov): see https://github.com/flutter/flutter/issues/138906
+  //                This test tests multi-view mode only, but currently there's
+  //                no way to cleanly boot into pure multi-view mode. As a
+  //                workaroud we boot into a mode that includes the implicit
+  //                view, but we then remove the implicit view.
+  await bootstrapAndRunApp();
+  final EngineFlutterWindow implicitView = EnginePlatformDispatcher.instance.implicitView!;
+  EnginePlatformDispatcher.instance.viewManager.disposeAndUnregisterView(implicitView.viewId);
```

Comment:
Mouad has landed the bit of code that disables the implicit view when `multiViewEnabled` is true. He's writing other tests for that, so he might be able to revisit this one!